src/nrc/setup_configuration/site.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.sites.models import Site
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+
+from .models import SiteConfigModel
+
+
+class SiteConfigurationStep(BaseConfigurationStep[SiteConfigModel]):
+    """
+    Configure the application site/domain.
+
+    **NOTE:** Site configuration will be depreciated
```

Comment:
Shouldn't this be upgraded (or disabled)? See [thread](https://github.com/maykinmedia/objects-api/pull/489#discussion_r1877969763)

Reply:
I will copy this from your PR in objecttypes once that is merged, to make sure they are the same

Reply:
I've removed it and will add an issue to add it in another PR